api/Document.json

```diff
@@ -7660,7 +7660,7 @@
               "version_added": "56"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
This is `document.onvisibilitychange`, where the data claimed that it was in IE 10.

https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilitychange_event#Browser_compatibility (just edited) says the event itself was in IE 10. I'm not sure which is correct, but can you back out this change and update both separately? The event can't be tested by mdn-bcd-collector, and it's just as well to separate out such changes.

Reply:
It looks like IE 10 didn't support `onvisibilitychange`, however it supported the event through `addEventListener`.  Reverting these changes and cherry-picking in a separate PR.

Reply:
Cherry-picked as #6833.